Assets/Scripts/Statistics/PatrollingVisualizer.cs

```diff
@@ -0,0 +1,53 @@
+
+using System.Collections.Generic;
+using Maes.Map;
+using Maes.Map.MapGen;
+using UnityEngine;
+
+namespace Maes.Statistics {
+    public class PatrollingVisualizer : MonoBehaviour {
+
+        public GameObject VertexVisualizer;
+
+        public GameObject EdgeVisualizer;
+
+        private PatrollingMap _patrollingMap;
+
+        private List<GameObject> _visualizers = new List<GameObject>();
+
+        public void SetMap(SimulationMap<Tile> simulationMap)
+        {
+            // Why the fuck do i have to do this?
```

Comment:
I think we need to either remove the comment or write something else :)

Reply:
Better?